frontend/src/metabase/components/ArchivedItem.jsx

```diff
@@ -24,7 +24,10 @@ const ArchivedItem = ({
   onToggleSelected,
   showSelect,
 }) => (
-  <div className="flex align-center p2 hover-parent hover--visibility border-bottom bg-light-hover">
+  <div
+    className="flex align-center p2 hover-parent hover--visibility border-bottom bg-light-hover"
+    data-testid={`archive-item-${name}`}
```

Comment:
Should we use `aria-label` instead of `data-testid`?

Reply:
That'd be preferred, but the change is meant to be backported. That means, it shouldn't have new user-facing strings and I'm not sure we have anything suitable available, so I'm using test ID for now

Reply:
Makes sense, thanks for the explaination